common/instructions.coffee

```diff
@@ -35,7 +24,7 @@ module.exports =
 		<br>
 		Make 100% sure it is the correct drive and click “Write”.
 		<br>
-		<strong>Warning!</strong>: Please make sure you have backed-up your SD card first,
+		<strong role="label">Warning!</strong>: Please make sure you have backed-up your SD card first,
```

Comment:
What's with the `role="label"` that were added on all `<strong>` tags?
Couldn't find it [in the spec](https://www.w3.org/TR/wai-aria-1.1/#x5-4-definition-of-roles) even though I'm not an expert on that.
Would it make sense to use something different, like `alert`
Will we handle them in the UI?

Reply:
Good idea, lets use `role="alert"`. In the UI we can handle them using a attribute selector.